lib/utils/logger.js

```diff
@@ -90,42 +90,123 @@ const methods = {
 
 Object.freeze(methods);
 
+function timestamp() {
+	return moment().format('MMM D HH:MM:SS');
```

Comment:
timestamp was configurable with cli args or rc config before

Reply:
Ok, will fix that